bciers/apps/administration/app/components/contacts/ContactsDataGrid.tsx

```diff
@@ -46,13 +46,22 @@ const ContactsDataGrid = ({
     [SearchCell, isExternalUser],
   );
 
+  // the mui grid requires a unique id, and since we can multiple rows with the same contact for different operators, we can't use the contact's id alone
+  function getRowId(row: ContactRow) {
```

Comment:
@Sepehr-Sobhani , FYI I just learned about [this prop](https://mui.com/x/react-data-grid/row-definition/?srsltid=AfmBOoqOYFCcKdcLzSipS6QXGGlQITIcWO6fE-YNcPONj-GYtu3LEGk0#:~:text=By%20default%2C%20the,where%20it%27s%20located.). We could have used it in the transfers grid instead of autogenerating a random uuid

Reply:
That's great, Thanks for sharing this! I can replace it with the existing code since I'm working on that grid too 😃